.openshift-ci/gcp/create-vm.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -e
+
+createGCPVM() {
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_IMAGE_PROJECT="$1"
+    shift
+    local GCP_IMAGE_FAMILY="$1"
+    shift
+    local GCP_SERVICE_ACCOUNT="$1"
+    shift
+
+    [ -z "$GCP_VM_NAME" ] && \
+            echo "error: missing parameter GCP_VM_NAME" && return 1
+    [ -z "$GCP_IMAGE_FAMILY" ] && \
+        echo "error: missing parameter GCP_IMAGE_FAMILY" && return 1
+    [ -z "$GCP_IMAGE_PROJECT" ] && \
+        echo "error: missing parameter GCP_IMAGE_PROJECT" && return 1
+    [ -z "$GCP_SERVICE_ACCOUNT" ] && \
+        echo "error: missing parameter GCP_SERVICE_ACCOUNT" && return 1
+
+    success=false
+    if gcloud compute instances create \
+        --image-family "$GCP_IMAGE_FAMILY" \
+        --image-project "$GCP_IMAGE_PROJECT" \
+        --service-account="$GCP_SERVICE_ACCOUNT" \
+        --scopes="storage-rw,cloud-platform"\
+        --machine-type e2-standard-2 \
+        --labels="stackrox-kernel-crawler-osci=true,stackrox-osci-job=${JOB_ID}" \
+        --boot-disk-size=20GB \
+        "$GCP_VM_NAME"; then
+        success=true
+        #break
+    else
+        gcloud compute instances delete "$GCP_VM_NAME"
+    fi
+
+    if test ! "$success" = "true"; then
+        echo "Could not boot instance."
+        return 1
+    fi
+
+    gcloud compute instances add-metadata "$GCP_VM_NAME" --metadata serial-port-logging-enable=true
+    gcloud compute instances describe --format json "$GCP_VM_NAME"
+    echo "Instance created successfully: $GCP_VM_NAME"
+
+    return 0
+}
+
+main() {
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_VM_TYPE="$1"
+    shift
+    local GCP_IMAGE_FAMILY="$1"
+    shift
+    local GCP_SERVICE_ACCOUNT="$1"
+    shift
+
+    which gcloud || true
+
+    echo "Creating the VM..."
+    createGCPVM \
+        "$GCP_VM_NAME"\
+        "$GCP_VM_TYPE"\
+        "$GCP_IMAGE_FAMILY"\
+        "$GCP_SERVICE_ACCOUNT"
+}
+
+main \
+    "kernel-packer-osci" \
```

Comment:
We probably want to do something to ensure the VM name is unique for each run, otherwise running from master and a PR concurrently would cause one of the runs to fail.

Reply:
Yep, that's planned as a follow-up step.